recipe/meta.yaml

```diff
@@ -68,8 +68,10 @@ requirements:
     - fmt
     - libsolv-static >=0.7.24
     - yaml-cpp-static              # [unix]
-    - libcurl >=7.88.1,<8          # [unix]
-    - libcurl-static >=7.88.1,<8   # [unix]
+    - libcurl >=8.4.0              # [linux]
+    - libcurl-static >=8.4.0       # [linux]
+    - libcurl <8.0                 # [osx]
+    - libcurl-static <8.0          # [osx]
```

Comment:
Why stay on `<8` on osx? Aside from the fact that those are still exposed to a bunch more CVEs (including the recent heap overflow that 8.4.0 fixes), v8 is ABI- & API-compatible with v7.

Reply:
The only point is that it doesn't build, so merging Linux now and investigating OSX.